specificatie/openapi.yaml

```diff
@@ -159,6 +159,66 @@ paths:
         default:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/default'
 
+  #19, 306, 373 
+  /adressen/_zoek:
```

Comment:
Ik zie dit liever als POST /adressen (dus zonder _zoek)

Reply:
OK

Reply:
we hebben dit in het team nog even besproken. Een aparte subresource met werkwoord als naam is hier prima om onderscheid te maken met standaard betekenis van POST (create).

Wel hebben we dan liever /{resource}/zoek, dus zonder de underscore. Die voldoet dan aan wat wordt voorgesteld in https://github.com/Geonovum/KP-APIs/issues/273, aan https://publicatie.centrumvoorstandaarden.nl/api/adr/#api-05 en https://publicatie.centrumvoorstandaarden.nl/api/adr/#api-10 (3e optie) en sluit aan bij ons bestaande endpoint GET /adressen/zoek

Reply:
Ik pas het aan.